Pipfile

```diff
@@ -4,14 +4,36 @@ verify_ssl = true
 name = 'pypi'
 
 [packages]
-django = "==2.2.7"
```

Comment:
Why did you change the pipfile?

Reply:
I didn't mean to change the pipfile

Reply:
https://www.atlassian.com/git/tutorials/rewriting-history

Here's some info on rebasing bad changes that made it into a PR.